### PR TITLE
feat(bigtable): add column family type to FamilyInfo in TableInfo

### PR DESCRIPTION
**Pull request by steveniemitz**
This adds ValueType to FamilyInfo and populates it when calling TableInfo.

Internally it adds methods to convert from protobuf Type to our own type model.

### CODE REVIEW

File: bigtable/type.go [hunk 1]
```diff
@@ -127,3 +145,59 @@ func (agg AggregateType) proto() *btapb.Type {
...
+
+func int64ProtoToType(i *btapb.Type_Int64) Type {
+	return Int64Type{Encoding: int64EncodingProtoToEncoding(i.Encoding)}
+}
+
+func aggregateProtoToType(agg *btapb.Type_Aggregate) Type {
+	it := protoToType(agg.InputType)
+
+	var aggregator Aggregator
+	switch agg.Aggregator.(type) {
```
Comment (bhshkh) [inline bigtable/type.go:196]: nil checks missing on agg and agg.Aggregator

File: bigtable/type.go [hunk 2]
```diff
@@ -127,3 +145,59 @@ func (agg AggregateType) proto() *btapb.Type {
...
+		return unknown[btapb.Type_Bytes_Encoding]{wrapped: be}
+	}
+}
+
+func bytesProtoToType(b *btapb.Type_Bytes) BytesType {
+	return BytesType{Encoding: bytesEncodingProtoToType(b.Encoding)}
+}
+
+func int64EncodingProtoToEncoding(ie *btapb.Type_Int64_Encoding) Int64Encoding {
+	switch e := ie.Encoding.(type) {
```
Comment (bhshkh) [inline bigtable/type.go:180]: nil checks missing on ie and ie.encoding

File: bigtable/type.go [hunk 3]
```diff
@@ -112,6 +122,14 @@ func (sum SumAggregator) fillProto(proto *btapb.Type_Aggregate) {
 	proto.Aggregator = &btapb.Type_Aggregate_Sum_{Sum: &btapb.Type_Aggregate_Sum{}}
 }
 
+type unknownAggregator struct {
+	wrapped *btapb.Type_Aggregate
+}
+
+func (ua unknownAggregator) fillProto(proto *btapb.Type_Aggregate) {
+	proto.Aggregator = ua.wrapped.Aggregator
```
Comment (bhshkh) [inline bigtable/type.go:130]: pointer nil check

File: bigtable/type.go
Comment (bhshkh) [inline bigtable/type.go:1]: Please add nil checks wherever needed

**Comment by bhshkh** [comment]
Please add integration tests

File: bigtable/type.go (hunk 3 above)
Comment (steveniemitz) [reply bigtable/type.go:130]: having basically no experience writing go, what am I supposed to do if its nil?

**Comment by bhshkh** [comment]
This has been already added in https://github.com/googleapis/google-cloud-go/pull/10520

Please reopen if something more needs to be added

**Pull request closed without merging**